src/audio/tdfb/tdfb.c

```diff
@@ -116,14 +122,37 @@ static void tdfb_free_delaylines(struct tdfb_comp_data *cd)
 		fir[i].delay = NULL;
 }
 
+static void *tdfb_filter_seek(struct sof_tdfb_config *config, int num_filters)
```

Comment:
What type is the coeff data, getting confused as we are using uint16 and uint8 here ?

Reply:
@lgirdwood Filter coefficients are int16_t. It's a bank of FIR filters, same that FIR EQ uses.

Reply:
let's make the return type `int16_t *` then?